src/models/Car/index.js

```diff
@@ -19,7 +20,14 @@ export class Car {
     this.#name = name.trim();
   }
 
+  #canMove() {
+    return randomInt(0, 10) >= 4;
+  }
+
   move() {
+    if (!this.#canMove()) {
+      return;
+    }
     this.#location += 1;
```

Comment:
저는 작업의 의미 단위로 코드 내에서도 개행을 한 번 더 주는 편입니다 :) 이 방법도 고려하신다면 훨씬 가독성 있는 코드를 작성하실 수 있을 거예요.

```suggestion

    this.#location += 1;
```